lib/elements/element.py

```diff
@@ -76,6 +77,147 @@ def __init__(self, node):
     def id(self):
         return self.node.get('id')
 
+    @property
+    @param('ties',
+           _('Allow lock stitches'),
+           tooltip=_('Tie thread at the beginning and/or end of this object. Manual stitch will not add lock stitches.'),
+           type='dropdown',
+           # Ties: 0 = Both | 1 = Before | 2 = After | 3 = Neither
+           # L10N options to allow lock stitch before and after objects
+           options=[_("Both"), _("Before"), _("After"), _("Neither")],
+           default=0,
+           sort_index=50)
+    @cache
+    def ties(self):
+        return self.get_int_param("ties", 0)
+
+    @property
+    @param('force_lock_stitches',
+           _('Force lock stitches'),
+           tooltip=_('Sew lock stitches after sewing this element, '
+                     'even if the distance to the next object is shorter than defined by the collapse length value in the Ink/Stitch preferences.'),
+           type='boolean',
+           default=False,
+           sort_index=51)
+    @cache
+    def force_lock_stitches(self):
+        return self.get_boolean_param('force_lock_stitches', False)
+
+    @property
+    @param('lock_start',
+           _('Lock start type'),
+           tooltip=_('Type of tack stitches.'),
+           type='dropdown',
+           options=[TIES[lock]["name"] for lock in LOCK_TYPES['start']],
+           default=0,
+           sort_index=52)
+    @cache
+    def lock_start(self):
+        return self.get_int_param('lock_start', 0)
+
+    @property
+    @param('lock_custom_start',
+           _('Custom svg path'),
+           tooltip=_("Enter a custom svg path. The last node will not be embroidered, but resembles the first stitch of the element."),
```

Comment:
I think maybe the word "represents" might go better here, rather than "resembles"?  I'm not sure though.